src/entities/community/ui/RecommendedPost.tsx

```diff
@@ -1,12 +1,105 @@
 import styled from '@emotion/styled';
+import moment from 'moment';
+moment.locale('ko');
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { boardPopUpDto } from 'entities/community/model/types';
 
 import gradient from 'shared/assets/image/community/hover-gradient.png';
 import arrow2 from 'shared/assets/image/main/arrow 2.png';
-import image1 from 'shared/assets/image/main/image1.png';
+import { ROUTES } from 'shared/config/routes';
 import { theme } from 'shared/styles/theme';
 import { Button } from 'shared/ui/';
 import { StarRatingInput } from 'shared/ui/Input/StarRatingInput';
 
+interface boardPopUpDtoProps {
+  boardPopUpDto: boardPopUpDto;
+}
+
+const RecommendedPost = ({ boardPopUpDto }: boardPopUpDtoProps) => {
+  const [recommendBoardFirstDto] = useState(boardPopUpDto.recommendBoardFirstDto);
```

Comment:
props에서 내려온 값을 사용하는 거라면 이렇게 해도 충분할 것 같습니다~ 

```suggestion
  const recommendBoardFirstDto = boardPopUpDto.recommendBoardFirstDto;
```